DirectProgramming/DPC++FPGA/Tutorials/Features/fp_pragmas/src/fp_pragmas.cpp

```diff
@@ -0,0 +1,259 @@
+//==============================================================
```

Comment:
Please confirm you ran clang-format on this file?

Reply:
Yes I did